### PR TITLE
Push pageview properties to the datalayer

### PR DESCRIPTION
**Pull request by vannio**
For tracking custom dimensions or metrics on GTM

### CODE REVIEW

File: src/lmn-gtm-analytics.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ const lmnAnalytics = {
       if (callback) {
         callback();
       }
+      dataLayer.push(properties);
```
Comment (barneyfoxuk) [inline src/lmn-gtm-analytics.js:81]: Can we make sure properties is never a function at this point please

**Review by barneyfoxuk — CHANGES_REQUESTED** [review]

File: src/lmn-gtm-analytics.js [hunk 2]
```diff
@@ -78,6 +78,9 @@ const lmnAnalytics = {
       if (callback) {
```
Comment (djgrant) [inline src/lmn-gtm-analytics.js:78]: Indentation seems to have gone off here. Mind fixing it while you're in the code?

File: src/lmn-gtm-analytics.js [hunk 3]
```diff
@@ -78,6 +78,9 @@ const lmnAnalytics = {
...
         callback();
       }
+      if (typeof properties !== 'function') {
```
Comment (djgrant) [inline src/lmn-gtm-analytics.js]: `properties` will never be a function at this point in the script.

**Review by djgrant — CHANGES_REQUESTED** [review]

File: src/lmn-gtm-analytics.js (hunk 2 above)
Comment (vannio) [reply src/lmn-gtm-analytics.js:78]: Not sure what you mean, it looks ok to me!

**Review by djgrant — APPROVED** [review]

File: src/lmn-gtm-analytics.js (hunk 2 above)
Comment (djgrant) [reply src/lmn-gtm-analytics.js:78]: No worries!

**Review by barneyfoxuk — APPROVED** [review]

**Pull request merged**